logging/sinks.js

```diff
@@ -181,24 +182,24 @@ cli
       description: 'The type of destination.'
     }
   }, function (options) {
-    program.createSink(options, console.log);
+    program.createSink(utils.pick(options, ['name', 'destination', 'filter', 'type']), utils.makeHandler(false));
   })
   .command('get <name>', 'Get the metadata for the specified sink.', {}, function (options) {
-    program.getSinkMetadata(options.name, console.log);
+    program.getSinkMetadata(options.name, utils.makeHandler());
   })
   .command('list', 'List all sinks in the authenticated project.', {}, function () {
-    program.listSinks(console.log);
+    program.listSinks(utils.makeHandler(true, 'id'));
   })
   .command('update <name> <metadata>', 'Update the metadata for the specified sink.', {}, function (options) {
     try {
       options.metadata = JSON.parse(options.metadata);
     } catch (err) {
       return console.error('"metadata" must be a valid JSON string!');
     }
-    program.updateSink(options, console.log);
+    program.updateSink(utils.pick(options, ['name', 'metadata']), utils.makeHandler(false));
```

Comment:
Another option (for later use): instead of _whitelisting_ the options we want, we _blacklist_ the ones we don't. (IIRC we briefly discussed this yesterday but didn't rule it out.)